helpers/resourceHints.js

```diff
@@ -33,7 +33,7 @@ const factory = globals => {
             }
         });
 
-        return new globals.handlebars.SafeString(_.map(hosts, format).join(''));
+        return new globals.handlebars.SafeString(_.map(hosts, host => format(host)).join(''));
     };
```

Comment:
Do we need the lodash map or can use the JS built in map on arrays?

Reply:
We can use the JS built in map but the links listed above show there being no performance difference.  I changed the syntax here as one of the links listed above shows this being faster than using the shorthand string iterator.

Reply:
I'm not worried about performance of the map function as I assume lodash to be fast and native to be as fast or faster.  I am more looking to reduce the reliance on outside dependencies like lodash where we can.  If we can use more vanilla js it should allow for easier maintenance as we don't have to worry as much about library version changes, security issues and so on.  I am not pushing to remove lodash completely, but where there are simple changes I'm hoping we can make them when supported by native language features.

@junedkazi @mattolson Please let me know if the above approach is not in alignment with the teams approach

I can't speak to the performance gain in regards to latency/first paint as I am still working on my understanding of how these projects are used in regards to a web consumer.  If this becomes part of the bundle to the user there will be performance increases as we will transfer smaller files with less code.